### DISCUSSION TITLE
Welcome to RepeatOBserver Discussions!

### DISCUSSION THREAD

**Discussion by celphin** [celphin/RepeatOBserverV1#1] [General]
## 👋 Welcome!
Please post here if you have any questions, comments or suggestions about using the code or interpreting the output.

**Comment by foreignsand** [comment]
i'm curious to know what the columns in the *_Centromere_histograms_summary file are. the first is the chromosome identifier, the second appears to be the prediction of the centromere location but i'm not sure what the third column is...
